Assets/Scripts/Entity/Player/Player.cs

```diff
@@ -19,7 +20,7 @@ public class Player : Entity
     void Awake()
     {
         SetEnabled(true);
-        SetWeapon(weapon);
+        //SetWeapon(weapon);
```

Comment:
You can just delete this line

Reply:
The SetWeapon, and all weapon logic, should now live in the ShootController. I simply commented it out to reduce any conflicts and allow for a refactor later.

Reply:
Ahh, right right